SU-SWS/stanford_media
# Hsd8 296 media validation

# READY FOR REVIEW

# Summary
- Added validation to check if files are similar when adding media items.

# Needed By (Date)
- 11/21

# Urgency
- medium

# Steps to Test

1. using suhumsci codebase sync to an existing site (don't checkout this branch til next step
1. checkout this branch
1. run database updates
1. validate the submodule `media_duplicate_validation` has been enabled
1. run cron to populate the tables
1. in any piece of content and any field upload an image and a document
1. in another piece of content attempt to upload the same image and document
1. verify it suggests to use the previously uploaded items
1. choose to use one of the previous items
1. verify that only 1 of the each item exists on /admin/content/media
1. go back to create a piece of content and upload the same image and document
1. this time choose to "add New" when suggestions are provided
1. verify 2 of each item exists on /admin/content/media


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
